components/supervisor/pkg/supervisor/ssh.go

```diff
@@ -213,3 +214,37 @@ func writeSSHEnv(cfg *Config, envvars []string) error {
 
 	return nil
 }
+
+func configureSSHDefaultDir(cfg *Config) {
+	if cfg.RepoRoot == "" {
+		log.Error("cannot configure ssh default dir with empty repo root")
+		return
+	}
+	file, err := os.OpenFile("/home/gitpod/.bashrc", os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0o644)
```

Comment:
`~/.ssh/rc` with `-oPermitUserRC` not works, don't know why, so we choose `.bashrc` to do it